Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/TextToolIntegrationTest.java

```diff
@@ -490,7 +527,11 @@ private void checkTextBoxDimensions() {
 		boolean italic = italicToggleButton.isChecked();
 
 		String font = (String) fontSpinner.getSelectedItem();
-		float textSize = (float) DEFAULT_TEXT_SIZE_20 * textSizeMagnificationFactor;
+
+		String stringTextSize = ((String) textSizeSpinner.getSelectedItem());
+		String[] splitedTextSize = stringTextSize.split("p");
```

Comment:
`stringTextSize = stringTextSize.substring(0, stringTextSize.indexOf("px"));` is easier to read